### PR TITLE
feat: Scrape LA county policy manual website

### PR DESCRIPTION
**Pull request by yoomlam**
## Ticket

https://navalabs.atlassian.net/browse/DST-633

## Changes

Add new scraper for LA County policy manual website.
See more in [Slack thread](https://nava.slack.com/archives/C06DP498D1D/p1733784125575129)

## Testing
Use playwright to expand nav bar in order to get a list of pages to scrape:
```
cd app/src/ingestion/la_policy/scrape
pip install -r requirements.txt
python scrape_la_policy_nav_bar.py
```
This creates `la_policy_nav_bar.html`, required for the next step.

Then use scrapy:
```
cd ../..
DEBUG_SCRAPINGS=true python scrape_la_policy.py &> out.log
# Alternatively: `DEBUG_SCRAPINGS=true poetry run scrape-la-policy` in the `app/` folder

# Check for errors
grep "ERROR:" out.log

# Examine warnings, ignoring "MarkupResemblesLocatorWarning" ones
grep "WARNING:" out.log | grep -v "MarkupResemblesLocatorWarning"
```

Scraped html and resulting markdown files are under `scraped/`.

`pretty-la_policy_scrapings.json` is generated for use in chunking in another PR.

### CODE REVIEW

File: app/src/ingestion/scrapy_dst/spiders/la_policy_spider.py [hunk 1]
```diff
@@ -0,0 +1,668 @@
...
+
+    def parse_page(self, response: HtmlResponse) -> dict[str, str]:
+        "Parses content pages; return value is add to file set by scrape_la_policy.OUTPUT_JSON"
+        url = response.url
+        title = response.xpath("head/title/text()").get().strip()
+        self.logger.info("parse_page %s", url)
+
+        # 40-101_19_Extended_Foster_Care_Benefits.htm and 1210_Overview.htm have extra '\r\n' and
+        # results in missing spaces in markdown text -- `replace("\r\n", "")` fixes this :shrug:
+        soup = BeautifulSoup(response.body.decode("utf-8").replace("\r\n", " "), "html.parser")
```
Comment (yoomlam) [inline app/src/ingestion/scrapy_dst/spiders/la_policy_spider.py:88]: https://nava.slack.com/archives/C06ETE82UHM/p1734110254545869?thread_ts=1733867173.636949&cid=C06ETE82UHM

File: app/src/ingestion/la_policy/scrape/requirements.txt [hunk 2]
```diff
@@ -0,0 +1,2 @@
+install-playwright
+playwright
```
Comment (fg-nava) [inline app/src/ingestion/la_policy/scrape/requirements.txt:2]: Needed to install `scrapy` for this env when running `python scrape_la_policy.py`, as I did not already have it.

```suggestion
playwright
scrapy
```

Comment (yoomlam) [reply app/src/ingestion/la_policy/scrape/requirements.txt]: Thanks!

**Comment by fg-nava** [comment]
Looks good to me -- test failures are on `tests/src/test_ingest_bem_pdfs.py` (as expected) and will be resolved if we merge #159 first

**Review by fg-nava — APPROVED** [review]

**Pull request merged**